src/main/app/src/components/hakutulos/hakutulosKortit/KoulutusKortti.js

```diff
@@ -61,8 +61,14 @@ const KoulutusKortti = ({ koulutus }) => {
   const muiScreenSizeMinSm = useMediaQuery(MUI_BREAKPOINTS.MIN_SM);
 
   const kuvaus =
-    _.truncate(l.localize(koulutus?.kuvaus).replace(/<[^>]*>/gm, ''), { length: 255 }) ||
-    t('haku.ei_kuvausta');
+    _.truncate(
+      l
+        .localize(koulutus?.kuvaus)
+        .replaceAll('</li>', ',</li>')
+        .replace('.,</li>', '.</li>')
```

Comment:
Varmaan halutaan tähänkin replaceAll?
```suggestion
        .replaceAll('.,</li>', '.</li>')
```

Reply:
Korjattu